app/models/app.rb

```diff
@@ -0,0 +1,4 @@
+class App < ActiveRecord::Base
```

Comment:
Happy to do that, but we did all agree on this naming during our whiteboarding session.